CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+5. Tests verifying that the service class works as expected should be added in the `tests/internal` directory. The dedicated test file should follow the same naming convention as the service class file, except it should have the `.js` extension. For example, if the service class file is named `systems-manager.ts`, the test file should be called `systems-manager.js`.
+6. Test files should consists in a k6 script using the [k6 chai js](https://k6.io/docs/javascript-api/jslib/k6chaijs/) library, and exporting a single `{serviceName}TestSuite(data)` function. This function should consist of a set of `describe` statements containing the actual test assertions, as demonstrated in the [existing s3 test suite](https://github.com/grafana/k6-jslib-aws/blob/main/tests/internal/s3.js). The test suite should be imported and called in the [`tests/internal/index.js`](https://github.com/grafana/k6-jslib-aws/blob/main/tests/index.js) test script, which is the entry point for the test suite.
+7. If the tests depend on a specific pre-existing state of the localstack setup, you can add a dedicated script in the `tests/internal/localstack_init` folder. Localstack will execute all the commands present in this script during its setup phase.
+8. The `npm test` command runs the test suite. This command will build the project and run the tests against the spun-up localstack docker container. The `docker-compose.yml` file contains the configuration for the container.
+9. Once the tests pass, the `src/index.ts` file should export the service class in the `src/index.ts` file so the user can use it.
+10. To get the build system to produce a dedicated 
+
+### Publishing a new version
+
+#### Expectations
+
+1. The service should have tests.
+2. The service should have documentation.
+3. The service should be re-exported in the `src/index.ts` file.
+4. The service should be exposed in the `aws.js` file in the `build` directory when running the `npm run webpack` command.
+5. The service should produce a dedicated `{service-name}.js` file in the `build` directory when running the `npm run webpack` command.
+6. The service should produce source map files for the dedicated `{service-name}.js` file and the `aws.js` file in the `build` directory when running the `npm run webpack` command.
+
+#### Steps
+
+1. Bump the version in the `package.json` file.
+2. Tag the latest main branch's commit with the new version.
+3. Create a dedicated GitHub version, and add the latest main branch's commit `build/` directory files.
```

Comment:
Is this referring to creating a tag for the new version under https://github.com/grafana/k6-jslib-aws/tags?

Reply:
Exactly 👍🏻 I've tried to make this more explicit indeed 🙇🏻